### PR TITLE
[FIX] Observatory in LE

### LINKED ISSUE #1701: Missing Observatory and War Hammer Nft in Expansion.
When placing items or reviewing my chest/inventory I noticed my observatory and war hammer nft don't show up anymore. Wanted to make sure we fix this as users will view that as missing nfts. 

Keep up the awesome work everyone 👏 ❤️

### PR DESCRIPTION
**Pull request by AzaroQL**
# Description

Added observatory as placeable in LE, added 5% xp boost.

https://user-images.githubusercontent.com/9656961/207430023-35824705-aea3-4717-8b35-19faf6a3353d.mp4

Fixes #1701 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manual test + yarn test

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AzaroQL** [comment]
TODO:
- Wait for art approval
- Wait for core team to add boost on BE
- Wait for core team to enable withdrawal on BE(or disable withdrawal on FE)

**Comment by stevew00dy** [comment]
I don't think I like the idea of the 5% XP boost for the item. It feels that it would be better to use it to increase the % of something better when expanding land. Maybe 5% better chance of rolling something rare. 

Just my thought as the observatory is about looking out and expansion feels like it connects better to that

**Comment by vrillx** [comment]
I would much rather it increases chance for an extra node when exploring new lands then the xp bonus. Thank you .

**Comment by Alexcnder** [comment]
I personally think a 5% exp gain is better then a 5% chance to get a tree or a plot

**Comment by TheDcol** [comment]
> I don't think I like the idea of the 5% XP boost for the item. It feels that it would be better to use it to increase the % of something better when expanding land. Maybe 5% better chance of rolling something rare.
> 
> Just my thought as the observatory is about looking out and expansion feels like it connects better to that

I remember back when it released it was said that it would give LE people 15% xp boost (or some kind of xp boost)

Given there are 5k observatories around, 5% xp boost seems reasonable.

The better roll chance for expansions would have been great if it was available day 1 😅
Yes there will be more expansions in future... But should people wait and not expand until the observatory is ready? A lot of the active beta players are onto the final expansion in 12 hours, not sure how long until other expansions are released

**Comment by adamhannigan** [comment]
Thanks for your patience! This is not forgotten - just pending developer implementation on backend 🙏  @AzaroQL

**Comment by AzaroQL** [comment]
Just ping me whenever BE is ready, Ill make a rebase.

**Comment by stevew00dy** [comment]
So the concept from my perspective was always an expansion boost - as the observatory looks out and that feels in line with expanding. I don't feel that an XP boost is well suited to this item but it can also work.

I know there is concern about the first few levels but initial plots have no RnG and the ones now which start to show some are very very early. When farms have 100+ plot sizes you wont worry about if the first 5 gave you the NFT boost.

**Comment by Amelavnyt** [comment]
Any reason why this hasn't been merged yet?

**Comment by Polysemouse** [comment]
> Any reason why this hasn't been merged yet?

Merge conflicts need to be resolved before it can be merged in.

File: src/features/game/events/landExpansion/feedBumpkin.test.ts
Comment (adamhannigan) [inline src/features/game/events/landExpansion/feedBumpkin.test.ts:102]: Cheeky `.only` to remove

**Review by adamhannigan — APPROVED** [review]
Looks good 👌 

The smaller observatory feels much nicer on the island

**Pull request merged**
